internal/config/config.go

```diff
@@ -30,7 +35,16 @@ type WorkerConfig struct {
 	MaxConcurrentSessions int
 }
 
-func (c Configuration) Validate() error { return nil }
+func (c Configuration) Validate() error {
+	var errs error
+
+	validate := validator.New(validator.WithRequiredStructEnabled())
```

Comment:
Can we create the validator inside the `Validate` function to avoid the dependency here for now?

Reply:
@jraddaoui oops, I didn't see your "thumbs up" until now. :facepalm: 

If we merge #5 first, then I can remove the `bagit.Config.Validate()` method altogether, and just run the validator here against the `bagit.Config` validation struct tag.

Reply:
Yeah, thumbs up is probably not the best way to provide feedback, sorry ;) Check my comments in the other PR.